fractal/parser.py

```diff
@@ -423,8 +431,14 @@
     argument_default=ap.SUPPRESS,
     allow_abbrev=False,
 )
-task_delete_parser.add_argument(
-    "task_id_or_name", help="ID or name of task to delete", type=str
+task_delete_id_or_name_group = task_delete_parser.add_mutually_exclusive_group(
+    required=True
+)
+task_delete_id_or_name_group.add_argument(
```

Comment:
Since this is a `task delete` command, I think we don't need the `task` prefix. The arguments should simply be `--id` and `--name` (and `--version`, if we want to have a consistent interface).

Reply:
Ref https://github.com/fractal-analytics-platform/fractal/pull/504/files#r1221010684